### PR TITLE
Add Billboard icon

### LINKED ISSUE #7885: Billboard Icon
### Brand Name

Billboard

### Website

https://www.billboard.com/

### Popularity Metric

The Similarweb global rank is 4506. See https://www.similarweb.com/website/billboard.com/#overview

### Official Resources for Icon and Color

svg: using svg-grabber from the official website
```
<svg xmlns="http://www.w3.org/2000/svg" style="height:100%;width:100%" viewBox="0 0 455.133 94.711"><path d="M70.107 28.859h21.318V93.69H70.107zM95.834 3.51h21.312v90.18H95.834zm25.723 0h21.312v90.18h-21.312zm250.711 32.035v-6.686h-21.315V93.69h21.315V62.792c0-7.949 4.154-12.237 11.728-12.237h2.522V28.229c-6.432.253-11.1 2.271-14.25 7.316zM80.764 0c-6.822 0-12.359 5.535-12.359 12.361 0 6.826 5.537 12.363 12.359 12.363 6.828 0 12.361-5.537 12.361-12.363S87.592 0 80.764 0zm244.457 33.795c-2.996-3.268-9.967-5.965-15.26-5.965-16.098 0-27.646 10.613-30.473 26.389-3.082-15.135-16.273-26.78-32.359-26.78-16.24 0-29.537 11.781-32.444 27.2-2.822-16.026-15.428-26.648-31.061-26.648-5.801 0-10.596 1.262-14.76 4.16V3.521h-21.312V93.7h20.938v-5.93c5.045 4.668 10.467 6.812 17.154 6.812 14.398 0 26.52-11.271 29.113-26.807 3.055 15.203 16.169 26.76 32.373 26.76 16.065 0 28.949-11.312 32.228-26.125 2.711 16.18 15.531 26.301 30.854 26.301 5.293 0 11.099-1.771 15.009-4.668V93.7h21.312V65.327c0-1.164-.008-2.298-.031-3.406.023-1.108.031-2.241.031-3.405V28.969h-21.312v4.826zM181.355 73.948c-6.826 0-12.359-5.535-12.359-12.363 0-6.824 5.533-12.358 12.359-12.358 6.824 0 12.359 5.534 12.359 12.358.001 6.828-5.534 12.363-12.359 12.363zm65.774-.99c-6.822 0-12.361-5.539-12.361-12.363 0-6.823 5.539-12.358 12.361-12.358 6.824 0 12.359 5.535 12.359 12.358.001 6.823-5.532 12.363-12.359 12.363zm64.047.392c-6.826 0-12.354-5.539-12.354-12.363 0-6.823 5.527-12.358 12.354-12.358 6.832 0 12.359 5.535 12.359 12.358.001 6.824-5.527 12.363-12.359 12.363zM433.815 3.5v28.631c-4.156-2.9-8.949-4.16-14.757-4.16-17.782 0-31.654 13.746-31.654 33.55 0 18.664 13.368 33.043 29.642 33.043 6.68 0 12.103-2.146 17.152-6.811v5.928h20.938V3.5h-21.321zm-12.553 70.288c-6.817 0-12.354-5.539-12.354-12.358 0-6.828 5.535-12.365 12.354-12.365 6.828 0 12.363 5.537 12.363 12.365 0 6.819-5.535 12.358-12.363 12.358zM36.072 27.99c-5.803 0-10.594 1.262-14.76 4.16V3.521H0V93.7h20.936v-5.93c5.045 4.668 10.469 6.812 17.156 6.812 16.268 0 29.637-14.383 29.637-33.047 0-19.797-13.874-33.545-31.657-33.545zm-2.269 45.958c-6.824 0-12.359-5.535-12.359-12.363 0-6.824 5.535-12.358 12.359-12.358 6.826 0 12.359 5.534 12.359 12.358 0 6.828-5.533 12.363-12.359 12.363z"></path></svg>
```
color: `#000000`, from the website & company logo from Facebook.

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Anmol-Baranwal**
billboard

**Issue:** closes #7885 

**Similarweb rank:**
Global rank: 4,506 &nbsp; `https://www.similarweb.com/website/billboard.com/#overview`

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG logo is retrieved from: https://www.billboard.com/
The hex value is extracted from the official website & company logo from Facebook.

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
May I please know, how much time it takes for the PRs to merge once all the checks have passed?

**Comment by LitoMore** [comment]
@Anmol-Baranwal There is no exact time. We maintain this project when we are free.

After all the checks have passed, we need to review your icon to ensure the processed shape perfectly fits the original shape.

For example this: #7752.

**Review by LitoMore — CHANGES_REQUESTED** [review]
CleanShot 2022-10-01 at 22 45 16@2x

@Anmol-Baranwal I see some differences. Could you re-process with precision 4? Use the command below:

```shell
npm run svgo -- -p 4 icons/billboard.svg
```

**Comment by Anmol-Baranwal** [comment]
Is this precision satisfactory?
preview_precision
With precision 3, below
preview_precision_3

**Comment by LitoMore** [comment]
CleanShot 2022-10-03 at 18 50 16@2x

@Anmol-Baranwal I still see some differences. Let's use this below directly:

```html
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Billboard</title><path d="M3.69685 11.02465h1.12413v3.41864H3.69685Zm1.35662-1.33669h1.12381v4.75533H5.05347Zm1.35641 0H7.5337v4.75533H6.40988Zm13.22037 1.68925v-.35256h-1.12397v3.41864h1.12397v-1.6293c0-.41916.21904-.64528.61843-.64528h.133v-1.17728c-.33918.01334-.58533.11975-.75143.38579zM4.2588 9.50287c-.35974 0-.65171.29187-.65171.65182 0 .35994.29197.65192.6517.65192.36006 0 .65182-.29198.65182-.65192 0-.35995-.29176-.65182-.65181-.65182zm12.89058 1.78207c-.15798-.17233-.52558-.31455-.80468-.31455-.84888 0-1.45782.55964-1.6069 1.39153-.1625-.79809-.8581-1.41215-1.70633-1.41215-.85636 0-1.55753.62123-1.71082 1.4343-.1488-.84507-.81354-1.40519-1.6379-1.40519-.30589 0-.55874.06655-.7783.21936v-1.5097H7.78063v4.75528h1.10409v-.3127c.26603.24615.55194.3592.90456.3592.75922 0 1.39843-.59433 1.53517-1.41357.1611.80168.85261 1.4111 1.70707 1.4111.84713 0 1.52653-.5965 1.69943-1.37761.14296.8532.81898 1.38689 1.62698 1.38689.2791 0 .58527-.0934.79145-.24615v.19284h1.12381v-1.49616a8.92817 8.92817 0 0 0-.0016-.1796 8.7632 8.7632 0 0 0 .0016-.17955v-1.55806H17.1494zm-7.58627 2.11732c-.35995 0-.6517-.29187-.6517-.65192 0-.35984.29175-.65165.6517-.65165.35984 0 .6517.29181.6517.65165.00006.36005-.2918.65192-.6517.65192zm3.46836-.0522c-.35973 0-.65181-.29208-.65181-.65192 0-.35978.29208-.65165.65181-.65165.35984 0 .6517.29187.6517.65165.00006.35979-.2917.65192-.6517.65192zm3.3773.02067c-.35995 0-.65145-.29208-.65145-.65192 0-.35978.29145-.65165.65145-.65165.36026 0 .6517.29187.6517.65165.00006.35984-.29144.65192-.6517.65192zm6.46693-3.6833v1.50976c-.21915-.15292-.4719-.21936-.77816-.21936-.93767 0-1.66916.72485-1.66916 1.76914 0 .98418.70492 1.7424 1.56307 1.7424.35224 0 .6382-.11315.90445-.35915v.3126H24V9.68743Zm-.66194 3.7064c-.35947 0-.65144-.29208-.65144-.65166 0-.36005.29187-.65202.65144-.65202.36005 0 .65192.29197.65192.65202 0 .35958-.29187.65166-.65192.65166zm-20.31164-2.415c-.306 0-.55864.06655-.77832.21936V9.68854H0v4.75528h1.10399v-.3127c.26603.24615.55204.3592.90466.3592.85783 0 1.5628-.75843 1.5628-1.74261 0-1.04393-.7316-1.76888-1.66932-1.76888zm-.11965 2.42343c-.35984 0-.6517-.29187-.6517-.65192 0-.35984.29186-.65165.6517-.65165.35995 0 .65171.29181.65171.65165 0 .36005-.29176.65192-.6517.65192z"/></svg>
```

I have already reviewed the difference:

CleanShot 2022-10-03 at 19 00 28@2x

**Comment by Anmol-Baranwal** [comment]
Do I have to use precision 5 on this processed one:
```
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Billboard</title><path d="M3.69685 11.02465h1.12413v3.41864H3.69685Zm1.35662-1.33669h1.12381v4.75533H5.05347Zm1.35641 0H7.5337v4.75533H6.40988Zm13.22037 1.68925v-.35256h-1.12397v3.41864h1.12397v-1.6293c0-.41916.21904-.64528.61843-.64528h.133v-1.17728c-.33918.01334-.58533.11975-.75143.38579zM4.2588 9.50287c-.35974 0-.65171.29187-.65171.65182 0 .35994.29197.65192.6517.65192.36006 0 .65182-.29198.65182-.65192 0-.35995-.29176-.65182-.65181-.65182zm12.89058 1.78207c-.15798-.17233-.52558-.31455-.80468-.31455-.84888 0-1.45782.55964-1.6069 1.39153-.1625-.79809-.8581-1.41215-1.70633-1.41215-.85636 0-1.55753.62123-1.71082 1.4343-.1488-.84507-.81354-1.40519-1.6379-1.40519-.30589 0-.55874.06655-.7783.21936v-1.5097H7.78063v4.75528h1.10409v-.3127c.26603.24615.55194.3592.90456.3592.75922 0 1.39843-.59433 1.53517-1.41357.1611.80168.85261 1.4111 1.70707 1.4111.84713 0 1.52653-.5965 1.69943-1.37761.14296.8532.81898 1.38689 1.62698 1.38689.2791 0 .58527-.0934.79145-.24615v.19284h1.12381v-1.49616a8.92817 8.92817 0 0 0-.0016-.1796 8.7632 8.7632 0 0 0 .0016-.17955v-1.55806H17.1494zm-7.58627 2.11732c-.35995 0-.6517-.29187-.6517-.65192 0-.35984.29175-.65165.6517-.65165.35984 0 .6517.29181.6517.65165.00006.36005-.2918.65192-.6517.65192zm3.46836-.0522c-.35973 0-.65181-.29208-.65181-.65192 0-.35978.29208-.65165.65181-.65165.35984 0 .6517.29187.6517.65165.00006.35979-.2917.65192-.6517.65192zm3.3773.02067c-.35995 0-.65145-.29208-.65145-.65192 0-.35978.29145-.65165.65145-.65165.36026 0 .6517.29187.6517.65165.00006.35984-.29144.65192-.6517.65192zm6.46693-3.6833v1.50976c-.21915-.15292-.4719-.21936-.77816-.21936-.93767 0-1.66916.72485-1.66916 1.76914 0 .98418.70492 1.7424 1.56307 1.7424.35224 0 .6382-.11315.90445-.35915v.3126H24V9.68743Zm-.66194 3.7064c-.35947 0-.65144-.29208-.65144-.65166 0-.36005.29187-.65202.65144-.65202.36005 0 .65192.29197.65192.65202 0 .35958-.29187.65166-.65192.65166zm-20.31164-2.415c-.306 0-.55864.06655-.77832.21936V9.68854H0v4.75528h1.10399v-.3127c.26603.24615.55204.3592.90466.3592.85783 0 1.5628-.75843 1.5628-1.74261 0-1.04393-.7316-1.76888-1.66932-1.76888zm-.11965 2.42343c-.35984 0-.6517-.29187-.6517-.65192 0-.35984.29186-.65165.6517-.65165.35995 0 .65171.29181.65171.65165 0 .36005-.29176.65192-.6517.65192z"/></svg>
```
or use precision 5 on the svg from scratch ?

Uptill now, I have been doing it from scratch, and both of the above gives a slight different result.

File: icons/billboard.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....1625-.798-.858-1.4121-1.7063-1.4121-.8564 0-1.5575.6212-1.7108 1.4343-.1488-.8451-.8136-1.4052-1.638-1.4052-.3058 0-.5587.0665-.7782.2193V9.6884H7.7806v4.7553h1.1041v-.3127c.266.2462.552.3592.9046.3592.7592 0 1.3984-.5943 1.5352-1.4135.161.8016.8526 1.411 1.707 1.411.8472 0 1.5266-.5965 1.6995-1.3776.143.8532.819 1.387 1.627 1.387.279 0 .5852-.0935.7914-.2462v.1928h1.1238v-1.4961a8.094 8.094 0 0 0-.0017-.1796 8.7508 8.7508 0 0 0 .0017-.1796v-1.558h-1.1238zM9.563 13.4022a.6518.6518 0 1 1 .0002-1.3036.6518.6518 0 0 1-.0002 1.3036zm3.4684-.0522a.6522.6522 0 0 1-.6518-.652.6518.6518 0 1 1 .6518.652zm3.3773.0206a.6517.6517 0 0 1-.6515-.6519.6516.6516 0 1 1 1.3032 0 .6517.6517 0 0 1-.6517.652zm6.467-3.6833v1.5098c-.2192-.153-.472-.2194-.7782-.2194-.9377 0-1.6692.7249-1.6692 1.7692 0 .9842.705 1.7424 1.563 1.7424.3523 0 .6383-.1132.9045-.3592v.3126H24V9.6873Zm-.662 3.7064a.652.652 0 0 1 0-1.3037c.36 0 .6519.292.6519.652a.652.652 0 0 1-.652.6517zM1.902 10.9787c-.306 0-.5586.0666-.7783.2194V9.6884H0v4.7553h1.104v-.3127c.266.2462.552.3592.9046.3592.8579 0 1.5629-.7584 1.5629-1.7426 0-1.044-.7316-1.7689-1.6694-1.7689zm-.1196 2.4235a.6519.6519 0 0 1 0-1.3036.6518.6518 0 0 1 0 1.3036z"/></svg>
```
Comment (LitoMore) [inline icons/billboard.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Billboard</title><path d="M3.69685 11.02465h1.12413v3.41864H3.69685Zm1.35662-1.33669h1.12381v4.75533H5.05347Zm1.35641 0H7.5337v4.75533H6.40988Zm13.22037 1.68925v-.35256h-1.12397v3.41864h1.12397v-1.6293c0-.41916.21904-.64528.61843-.64528h.133v-1.17728c-.33918.01334-.58533.11975-.75143.38579zM4.2588 9.50287c-.35974 0-.65171.29187-.65171.65182 0 .35994.29197.65192.6517.65192.36006 0 .65182-.29198.65182-.65192 0-.35995-.29176-.65182-.65181-.65182zm12.89058 1.78207c-.15798-.17233-.52558-.31455-.80468-.31455-.84888 0-1.45782.55964-1.6069 1.39153-.1625-.79809-.8581-1.41215-1.70633-1.41215-.85636 0-1.55753.62123-1.71082 1.4343-.1488-.84507-.81354-1.40519-1.6379-1.40519-.30589 0-.55874.06655-.7783.21936v-1.5097H7.78063v4.75528h1.10409v-.3127c.26603.24615.55194.3592.90456.3592.75922 0 1.39843-.59433 1.53517-1.41357.1611.80168.85261 1.4111 1.70707 1.4111.84713 0 1.52653-.5965 1.69943-1.37761.14296.8532.81898 1.38689 1.62698 1.38689.2791 0 .58527-.0934.79145-.24615v.19284h1.12381v-1.49616a8.92817 8.92817 0 0 0-.0016-.1796 8.7632 8.7632 0 0 0 .0016-.17955v-1.55806H17.1494zm-7.58627 2.11732c-.35995 0-.6517-.29187-.6517-.65192 0-.35984.29175-.65165.6517-.65165.35984 0 .6517.29181.6517.65165.00006.36005-.2918.65192-.6517.65192zm3.46836-.0522c-.35973 0-.65181-.29208-.65181-.65192 0-.35978.29208-.65165.65181-.65165.35984 0 .6517.29187.6517.65165.00006.35979-.2917.65192-.6517.65192zm3.3773.02067c-.35995 0-.65145-.29208-.65145-.65192 0-.35978.29145-.65165.65145-.65165.36026 0 .6517.29187.6517.65165.00006.35984-.29144.65192-.6517.65192zm6.46693-3.6833v1.50976c-.21915-.15292-.4719-.21936-.77816-.21936-.93767 0-1.66916.72485-1.66916 1.76914 0 .98418.70492 1.7424 1.56307 1.7424.35224 0 .6382-.11315.90445-.35915v.3126H24V9.68743Zm-.66194 3.7064c-.35947 0-.65144-.29208-.65144-.65166 0-.36005.29187-.65202.65144-.65202.36005 0 .65192.29197.65192.65202 0 .35958-.29187.65166-.65192.65166zm-20.31164-2.415c-.306 0-.55864.06655-.77832.21936V9.68854H0v4.75528h1.10399v-.3127c.26603.24615.55204.3592.90466.3592.85783 0 1.5628-.75843 1.5628-1.74261 0-1.04393-.7316-1.76888-1.66932-1.76888zm-.11965 2.42343c-.35984 0-.6517-.29187-.6517-.65192 0-.35984.29186-.65165.6517-.65165.35995 0 .65171.29181.65171.65165 0 .36005-.29176.65192-.6517.65192z"/></svg>
```

I mean use my one directly.

Comment (Anmol-Baranwal) [reply icons/billboard.svg]: I have updated the SVG file

**Review by LitoMore — APPROVED** [review]

**Pull request merged**